src/GovUk.Education.ExploreEducationStatistics.Public.Data.Model/Extensions/DataSetVersionExtensions.cs

```diff
@@ -0,0 +1,11 @@
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Extensions;
+
+public static class DataSetVersionExtensions
+{
+    public static bool CanBeDeleted(this DataSetVersion dataSetVersion) => dataSetVersion.Status.IsDeletableState();
+
+    public static bool IsDeletableState(this DataSetVersionStatus status) => status is DataSetVersionStatus.Failed
```

Comment:
Why two methods? Just a single one is sufficient as they do the same thing.

Reply:
One is for the model, one is for the status. In the viewmodels for the deletion plan I didn't have access to the full model when calculating the `Valid` flag, because it was auto-calculated. So I needed to use the status

Reply:
After discussing, I've turned this into just the one method and moved it to the model itself. We no longer auto-calculate the `valid` flag, so can remove the extension methods